frontend/components/ClubEditPage.tsx

```diff
@@ -254,27 +255,6 @@ class ClubForm extends Component<ClubFormProps, ClubFormState> {
                   },
                 ]}
               />
-              <MembersCard club={club} />
-            </>
-          ),
-          disabled: !isEdit,
-        },
-        {
-          name: 'events',
-          label: 'Events',
-          content: (
-            <>
-              <EventsCard club={club} />
-            </>
```

Comment:
@snie11 This is undoing some of the changes that Davis made, might be a merge conflict issue.

Reply:
You can fix this easily by reverting the whole file to master.